Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -240,6 +292,13 @@ public bool InteractUsing(InteractUsingEventArgs eventArgs)
 
             if (!tool.UseTool(eventArgs.User, Owner, ToolQuality.Prying)) return false;
 
+            if (IsBolted())
+            {
+                var notify = IoCManager.Resolve<IServerNotifyManager>();
+                notify.PopupMessage(Owner, eventArgs.User, "The door won't budge!");
```

Comment:
Both tg and vg mention that it's because of the bolts, they both say:
`The airlock's bolts prevent it from being forced!`

I don't think there's anything to gain by not telling the player why it won't budge.

Reply:
Yeah. Forgot to check which string gets printed.